site_scons/site_tools/go_builder.py

```diff
@@ -63,21 +77,19 @@ def _check_go_version(context):
 
         context.Display(f'Checking {env.d_go_bin} version... ')
         cmd_rc = subprocess.run([env.d_go_bin, 'version'], check=True, stdout=subprocess.PIPE)
-        out = cmd_rc.stdout.decode('utf-8')
+        out = cmd_rc.stdout.decode('utf-8').strip()
         if len(out.split(' ')) < 3:
             context.Result(f'failed to get version from "{out}"')
             return 0
 
         # go version go1.2.3 Linux/amd64
-        go_version = out.split(' ')[2].replace('go', '')
-        if '-' in go_version:
-            go_version = go_version.split('-')[0]
+        go_version = get_go_version(out)
         if len([x for x, y in
                 zip(go_version.split('.'), MIN_GO_VERSION.split('.'))
                 if int(x) < int(y)]) > 0:
-            context.Result(f'{go_version} is too old (min supported: {MIN_GO_VERSION}) ')
+            context.Result(f'{out} is too old (min supported: {MIN_GO_VERSION}) ')
             return 0
-        context.Result(str(go_version))
```

Comment:
Please revert these changes. The scons output looks wacky if the raw output string is included.

Reply:
Did you see my other comment, we actually have no idea what the version was that was triggering the failure in the first place because it simply wasn't logged.

Reply:
Ah I see now, this is logged if the version can be parsed, it's the other change that's logged if it cannot be.